File: app/src/main/java/com/aaronbrecher/neverlate/network/AppApiUtils.java
package com.aaronbrecher.neverlate.network;

public class AppApiUtils {
    static final String DIRECTION_MATRIX_API_ENDPOINT = "direction-matrix";
    static final String VERSION_ENDPOINT = "current-version";
    static final String DIRECTION_API_ENDPOINT = "directions";
    static final String PUBLIC_TRANSIT_API_ENDPOINT = "public-transit";
    static final String VERIFY_PURCHASE_ENDPOINT = "verify-purchase";
    private static final String BASE_URL = "https://never-late-spring-api-staging.herokuapp.com/";

    public static AppApiService createService(){
        return RetrofitClient.getClient(BASE_URL).create(AppApiService.class);
    }
}

File: app/src/main/java/com/aaronbrecher/neverlate/network/AppApiUtils.kt
package com.aaronbrecher.neverlate.network


const val DIRECTION_MATRIX_API_ENDPOINT = "direction-matrix"
const val VERSION_ENDPOINT = "current-version"
const val DIRECTION_API_ENDPOINT = "directions"
const val PUBLIC_TRANSIT_API_ENDPOINT = "public-transit"
const val VERIFY_PURCHASE_ENDPOINT = "verify-purchase"
private const val BASE_URL = "https://never-late-spring-api-staging.herokuapp.com/"

fun createRetrofitService(): AppApiService {
    return getRetrofitClient(BASE_URL).create(AppApiService::class.java)
}
